### PR TITLE
fix:  missing styles of the iframe

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

**Review by EnoF — CHANGES_REQUESTED** [review]
The iframe shouldn't be dark, or at least I don't remember having everything having the right contrast in dark mode yet

File: spirekey/src/app/(wallet)/global.css.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import '@kadena/kode-ui/global';
 import { tokens, vars } from '@kadena/kode-ui/styles';
-import { globalStyle } from '@vanilla-extract/css';
+import { globalStyle, style } from '@vanilla-extract/css';
```
Comment (EnoF) [inline spirekey/src/app/(wallet)/global.css.ts:3]: Unused var?

File: spirekey/src/app/(wallet)/layout.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const BetaNotification = dynamic(
   },
 );
 
-import '../global.css';
+import './global.css';
```
Comment (EnoF) [inline spirekey/src/app/(wallet)/layout.tsx:15]: I don't see any global.css on this path?

Comment (Randynamic) [reply spirekey/src/app/(wallet)/layout.tsx:15]: Is this because we miss a prefix?

**Review by Randynamic — APPROVED** [review]

File: spirekey/src/app/(wallet)/layout.tsx (hunk 2 above)
Comment (EnoF) [reply spirekey/src/app/(wallet)/layout.tsx:15]: nvm, the file got moved in this pr :joy:

**Review by EnoF — APPROVED** [review]

**Pull request merged**